music/src/music_vae/model.ts

```diff
@@ -865,13 +867,30 @@ class MusicVAE {
     }
 
     if (this.chordEncoder) {
-      // TODO(iansimon): handle chords for variable-length segments
```

Comment:
I think it would be more readable to put this and the variable-length code above into separate private methods.

Reply:
Ideally it would also be used by the decoder.